source/community/conduct-guidelines.md

```diff
@@ -55,6 +59,6 @@ Changes to this document will be tracked within and reflected via its [internal
   | 2019-01-22 | Update IIIF reference to refer to any IIIF staff member             |
 
 [iiif-slack]: https://iiif.slack.com
-[iiif-conduct]: {{ site.url }}{{ site.baseurl }}/event/conduct/
+[iiif-conduct]: {{ site.url }}{{ site.baseurl }}/community/conduct/
 [change-log]: #change-log "Change Log"
-[guidelines]: {{ site.url }}{{ site.baseurl }}/event/conduct/#reporting-guidelines
+[guidelines]: {{ site.url }}{{ site.baseurl }}/community/conduct/#reporting-guidelines
```

Comment:
Should this be #reporting-harassment to match: https://github.com/IIIF/website/blob/wireframe/source/community/conduct.md#reporting-harassment